webpack.config.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+
+var webpack = require('webpack');
+
```

Comment:
How do you build demo?

Reply:
DEMO should reference the output in `dist`.

Reply:
Not sure if hot-reload still work... Should leave it as it is now and investigate later.